crates/configuration/src/version5/introspection.sql

```diff
@@ -1322,9 +1322,31 @@ WITH
     SELECT
       v ->> 'operatorName' AS operator_name,
       v ->> 'exposedName' AS exposed_name,
-      v ->> 'operatorKind' AS operator_kind
+      v ->> 'operatorKind' AS operator_kind,
+      null as operator_type,
+      null as operator_type_name
     FROM
       jsonb_array_elements($4) AS v
+    UNION ALL
+    SELECT DISTINCT ON (oprname)
+      o.oprname AS operator_name,
+      o.oprcode::text AS exposed_name,
+      'custom' AS operator_kind,
+      o.oprleft as operator_type,
+      (SELECT typname FROM pg_catalog.pg_type WHERE oid = o.oprleft LIMIT 1) as operator_type_name
+    FROM
+      pg_operator as o
+    WHERE
+      o.oprname IS NOT NULL
+      AND o.oprcode::text IS NOT NULL
+      AND o.oprleft = o.oprright
+      AND o.oprresult = (SELECT oid FROM pg_catalog.pg_type WHERE typname = 'bool' LIMIT 1)
```

Comment:
This code generates a result that looks like this:

![Screenshot from 2024-07-19 16-54-44](https://github.com/user-attachments/assets/6760b1e2-3b21-4783-9c93-81811a327701)

Reply:
Considering that this gets the `exposedName` from `pg_operator.oprcode`, which is a reference to `pg_proc.oid` (which when cast to text via `::text` yields the value `pg_proc.proname`) I should think that simply adding `network_supeq` to the list of known prefix-functions in the introspection options should make it appear (and `text_pattern_ge` and friends too)